next.mdx.use.mjs

```diff
@@ -38,6 +39,8 @@ export const mdxComponents = {
   DownloadLink: DownloadLink,
   // Renders a Button Component for `button` tags
   Button: Button,
+  // Renders a Search Page
+  SearchPage: SearchPage,
```

Comment:
Since this is only used in one place, it shouldn't be added here. You can import your SearchPage component from within the .mdx file using regular imports.

Also, I'd recommend calling the component `SearchResultsContainer` or something like that, and put it under `components/MDX/SearchResults` or something.

Reply:
cc @micheleriva just in case this was lost on the sea of feedback.

Reply:
It's on my todo list, I'm keeping this for last